packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -127,11 +129,20 @@ const ConnectorDetails = () => {
             <div>
               <ConnectorTypeName type={data.type} />
               <div className={styles.verticalBar} />
-              <Status status={data.enabled ? 'enabled' : 'disabled'} varient="outlined">
-                {t('connectors.connector_status', {
-                  context: data.enabled ? 'enabled' : 'disabled',
-                })}
-              </Status>
+              {data.type === ConnectorType.Social && inUse !== undefined && (
+                <Status status={inUse ? 'enabled' : 'disabled'} varient="outlined">
+                  {t('connectors.connector_status', {
+                    context: inUse ? 'in_use' : 'not_in_use',
+                  })}
+                </Status>
+              )}
+              {data.type !== ConnectorType.Social && (
+                <Status status={data.enabled ? 'enabled' : 'disabled'} varient="outlined">
```

Comment:
will we remove `enabled` in the future?

Reply:
YES. For now, `enabled` is used to simplfy the syncing between db and connector code(packages).